common/src/main/java/com/mapbox/vision/common/models/UiSign.kt

```diff
@@ -428,7 +428,20 @@ sealed class UiSign {
         AheadSpeedLimit(usResourceName = "ahead_speedlimit", hasNumber = true),
         WarningSpeedLimit(usResourceName = "warning_speedlimit", hasNumber = true),
         RegulatoryNoUTurnRight(usResourceName = "regulatory_no_u_turn_right"),
-        WarningTurnRightOnlyArrow(usResourceName = "warning_turn_right_only_arrow")
+        WarningTurnRightOnlyArrow(usResourceName = "warning_turn_right_only_arrow"),
+
+        InformationCarWashing,
```

Comment:
don't we have resources for these signs?

Reply:
no assets yet, see description